packages/key-management/src/util/mapHardwareSigningData.ts

```diff
@@ -87,6 +87,12 @@ const matchGroupedAddress = (knownAddresses: GroupedAddress[], outputAddress: Bu
   return knownAddresses.find(({ address }) => address === outputAddressBech32);
 };
 
+export const mintValueAsString = (scope: ManagedFreeableScope, amount: CML.Int): string => {
```

Comment:
I think we could use the `to_str()` directly here. [The docs](https://github.com/dcSpark/cardano-multiplatform-lib/blob/685354495562ff879101fd023eb89555db9bbb1e/rust/pkg/cardano_multiplatform_lib.js.flow#L3401) suggest it will add the `-` in anyway.
```diff
-  const positiveAmount = scope.manage(amount?.as_positive())?.to_str();
-  const negativeAmount = scope.manage(amount?.as_negative())?.to_str();
-  return positiveAmount ? positiveAmount : `-${negativeAmount}`;
+  return amount?.to_str();

Reply:
Great catch. Will update

Reply:
67e3f4aa61259851b735f3652c1fd95e64000848